src/cli/cli.cpp

```diff
@@ -2932,17 +2843,33 @@ void Interpreter::ProcessJoinerPort(int argc, char *argv[])
 }
 #endif
 
+void Interpreter::ProcessBlacklist(int argc, char *argv[])
+{
+    ProcessAddressFilter(OT_MAC_ADDRESSFILTER_MODE_BLACKLIST, argc, argv);
```

Comment:
Since we are keeping whitelist/blacklist spinel properties as before, I thought it might be easier understood to keep whitelist spinel-cli as before too. Here intended to keep soc cli consistent with spinel-cli.
Or you prefer to update both soc cli and spinel-cli to mirror the API updates?